libs/data-mapper/src/lib/mapDefinitions/__test__/MapDefinitionSerializer.spec.ts

```diff
@@ -1,1888 +1,3705 @@
-      const sourceNode = extendedSourceSchema.schemaTreeRoot.children[3];
-      const targetNode = extendedTargetSchema.schemaTreeRoot.children[4];
-      const ifFunctionId = createReactFlowFunctionKey(ifPseudoFunction);
-      const greaterThanId = createReactFlowFunctionKey(greaterThanFunction);
-      const mapDefinition: MapDefinitionEntry = {};
-      const connections: ConnectionDictionary = {};
-
-      // Source to greater than
-      setConnectionInputValue(connections, {
-        targetNode: greaterThanFunction,
-        targetNodeReactFlowKey: greaterThanId,
-        findInputSlot: true,
-        value: {
-          reactFlowKey: addReactFlowPrefix(sourceNode.children[0].key, SchemaType.Source),
-          node: sourceNode.children[0],
-        },
-      });
-      setConnectionInputValue(connections, {
-        targetNode: greaterThanFunction,
-        targetNodeReactFlowKey: greaterThanId,
-        findInputSlot: true,
-        value: {
-          reactFlowKey: addReactFlowPrefix(sourceNode.children[1].key, SchemaType.Source),
-          node: sourceNode.children[1],
-        },
-      });
-
-      // Inputs to conditional
-      setConnectionInputValue(connections, {
-        targetNode: ifPseudoFunction,
-        targetNodeReactFlowKey: ifFunctionId,
-        findInputSlot: true,
-        value: {
-          reactFlowKey: greaterThanId,
-          node: greaterThanFunction,
-        },
-      });
-      setConnectionInputValue(connections, {
-        targetNode: ifPseudoFunction,
-        targetNodeReactFlowKey: ifFunctionId,
-        findInputSlot: true,
-        value: {
-          reactFlowKey: addReactFlowPrefix(sourceNode.key, SchemaType.Source),
-          node: sourceNode,
-        },
-      });
-
-      //Conditional to target
-      setConnectionInputValue(connections, {
-        targetNode: targetNode,
-        targetNodeReactFlowKey: addReactFlowPrefix(targetNode.key, SchemaType.Target),
-        findInputSlot: true,
-        value: {
-          reactFlowKey: ifFunctionId,
-          node: ifPseudoFunction,
-        },
-      });
-
-      //Child property not connected to the conditional
-      setConnectionInputValue(connections, {
-        targetNode: targetNode.children[0],
-        targetNodeReactFlowKey: addReactFlowPrefix(targetNode.children[0].key, SchemaType.Target),
-        findInputSlot: true,
-        value: {
-          reactFlowKey: addReactFlowPrefix(sourceNode.children[0].key, SchemaType.Source),
-          node: sourceNode.children[0],
-        },
-      });
-      setConnectionInputValue(connections, {
-        targetNode: targetNode.children[1],
-        targetNodeReactFlowKey: addReactFlowPrefix(targetNode.children[1].key, SchemaType.Target),
-        findInputSlot: true,
-        value: {
-          reactFlowKey: addReactFlowPrefix(sourceNode.children[1].key, SchemaType.Source),
-          node: sourceNode.children[1],
-        },
+  describe('XML to XML', () => {
```

Comment:
From here to the next comment is unchanged, just slightly indented more since it is wrapped in an additional `describe()`